helm_deploy/values-prod.yaml

```diff
@@ -15,7 +15,8 @@ generic-service:
     WELCOME_API_URL: "https://welcome-api.prison.service.justice.gov.uk"
     HMPPS_COOKIE_NAME: hmpps-session-prod
     HMPPS_COOKIE_DOMAIN: welcome.prison.service.justice.gov.uk
-    ENABLED_PRISONS: ''
+    ENABLED_PRISONS: 'NMI'
+    CONFIRM_ENABLED: 'false'
```

Comment:
enabling Nottingham in production but with confirming disabled. 
Confirm disabled in preprod so should be able to test this in preprod before releasing (data permitting)